pandas/tests/io/excel/test_readers.py

```diff
@@ -1148,6 +1148,22 @@ def test_read_excel_skiprows(self, request, read_ext):
         )
         tm.assert_frame_equal(actual, expected)
 
+        actual = pd.read_excel(
```

Comment:
Please create a dedicated test

Reply:
I've split this to a new test.

Reply:
Wondering if the other "callable" test should be in there too, and we can call the test `test_read_excel_skiprows_callable`